README.md

```diff
@@ -531,8 +531,28 @@ firewalld::direct_passthroughs:
 * `args`: Name of the passthroughhrough to add (e.g: -A OUTPUT -j OUTPUT_filter)
 
 
+## Testing
```

Comment:
We've got our contributing guidelines which explain how to execute tests: https://github.com/voxpupuli/puppet-firewalld/blob/master/.github/CONTRIBUTING.md

That file is managed by modulesync: https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.github/CONTRIBUTING.md.erb

Could you link to that file / update it instead of duplicating the information?

Reply:
@bastelfreak The point here is that these tests have to be run in vagrant and not docker like we might otherwise.

Reply:
Can't test underlying firewalls properly in Docker-land